template.env

```diff
@@ -14,6 +14,7 @@ GROUPS=["103582791469033930"]
 ALERTS=["trade"]
 
 PRICESTF_API_TOKEN=""
+PRICESTF_URL=""
```

Comment:
maybe best to just change these two to other name, such as, `CUSTOM_PRICER_URL` and `CUSTOM_PRICER_API_TOKEN`

Reply:
Yes. Thanks for the feedback